src/screens/nano-contract/NanoContractDetail.js

```diff
@@ -51,25 +53,57 @@ function NanoContractDetail() {
   const navigate = useNavigate();
   const { nc_id: ncId } = useParams();
   const nc = nanoContracts[ncId];
-  let blueprintInformationAux = nc != null ? blueprintsData[nc.blueprintId] : null;
 
   // loading {boolean} Bool to show/hide loading element
   const [loading, setLoading] = useState(true);
   // data {Object} Nano contract loaded data
   const [data, setData] = useState(null);
   // blueprintInformation {Object} Blueprint information data
-  const [blueprintInformation, setBlueprintInformation] = useState(blueprintInformationAux);
+  const [blueprintInformation, setBlueprintInformation] = useState(null);
```

Comment:
Update docs right above this to `Object | null`

Reply:
Done https://github.com/HathorNetwork/hathor-wallet/pull/695/commits/70c70274e815e718dff72325557821dc65876b25